kiva/agg/tests/test_save.py

```diff
@@ -3,14 +3,12 @@
 
 from numpy import allclose, ravel
 
-import nose
-
 from kiva import agg
 
 
 # FIXME:
 #   These tests are broken, and Peter promised to fix it at some point.
-
+@unittest.skip("tests are broken")
```

Comment:
😂  Do you mind opening an issue and reference it here please?

Reply:
opened #480

Reply:
Thank you.
```suggestion
@unittest.skip("tests are broken. See enthought/enable#480")
```